manifest/xyz.jvyden/DynTunes/info.json

```diff
@@ -0,0 +1,24 @@
+{
+	"name": "DynTunes",
+	"id": "xyz.jvyden.DynTunes",
+	"description": "Add Dynamic Variables to show the music you're listening to",
+	"category": "Misc",
+	"sourceLocation": "https://github.com/jvyden/DynTunes",
+    "platforms": ["linux"],
+	"versions": {
+		"1.0.0": {
+			"releaseUrl": "https://github.com/jvyden/DynTunes/releases/tag/v1.0.0",
+			"artifacts": [
+                {
+				    "url": "https://github.com/jvyden/DynTunes/releases/download/v1.0.0/DynTunes.dll",
```

Comment:
Non-blocking: recommended to set mod version by string or constant. See example mod for a way to set assembly and mod info by changing in one spot.